openstack/identity/v2/tokens/results.go

```diff
@@ -23,6 +23,10 @@ type Token struct {
 
 	// Tenant provides information about the tenant to which this token grants access.
 	Tenant tenants.Tenant
+
+	// the owner user of token
+	UserName string
+	UserID   string
```

Comment:
I'm not sure we want these as part of the `Token`. Maybe instead we have an `ExtractUser` method?